# ループ周期 [Hz]
hz: 10

# デバッグ用
is_visible: false # 可視化フラグ(パス・ノード)
sleep_time: 0.005 # 次のpublishまでの時間 [s]

# 経由点 [m]
way_points_x: [0.0, 10.6, 11.4, 11.5, 11.6, 10.6, -20.8, -21.8, -21.9, -20.9, 0.0]
way_points_y: [0.0,  0.0,  0.8,  9.3, 13.5, 14.2,  14.2,  13.2,   1.2,   0.1, 0.0]
